com.unity.multiplayer.mlapi/Prototyping/NetworkTransform.cs

```diff
@@ -41,24 +40,20 @@ public enum Authority
         /// TODO MTT-766 once we have per var interpolation
         /// Enable interpolation
         /// </summary>
-        // ReSharper disable once NotAccessedField.Global
```

Comment:
Are these changes related to the parenting thing or just cleanup?

Reply:
cleanup. could be separated and I'll separate stuff that are not required to go under this PR.